daprdocs/content/en/operations/monitoring/metrics/metrics-overview.md

```diff
@@ -72,6 +72,30 @@ spec:
     enabled: false
 ```
 
+## High cardinality metrics
+
+Depending on your use case, some metrics emitted by Dapr might contain values that have a high cardinality. This might cause large memory usage for the Dapr process/container and incur expensive egress costs in certain cloud environments. To mitigate this issue, you can set regular expressions for every metric exposed by the Dapr sidecar. For a list of all metrics, see [this link](https://github.com/dapr/dapr/blob/master/docs/development/dapr-metrics.md).
```

Comment:
```suggestion
Depending on your use case, some metrics emitted by Dapr might contain values that have a high cardinality. This might cause increased memory usage for the Dapr process/container and incur expensive egress costs in certain cloud environments. To mitigate this issue, you can set regular expressions for every metric exposed by the Dapr sidecar. [See a list of all Dapr metrics](https://github.com/dapr/dapr/blob/master/docs/development/dapr-metrics.md).
```